components/WeeklyPickButton/WeeklyPickButton.test.tsx

```diff
@@ -15,10 +16,14 @@ describe('WeeklyPickButton', () => {
   it('renders correctly', () => {
     render(
       <RadioGroup>
-        <WeeklyPickButton team={weeklyPickData.team} src={weeklyPickData.src} />
+        <WeeklyPickButton homeAway={weeklyPickData.homeAway} team={weeklyPickData.team} src={weeklyPickData.src} />
```

Comment:
Need to write 2 separate tests for this. 1 to check that the team is home and 1 that checks the team is away.